rules/rollup.go

```diff
@@ -73,21 +73,36 @@ func NewRollupTargetFromFields(name string, tags []string, policies []policy.Pol
 	return RollupTarget{Name: []byte(name), Tags: bytesArrayFromStringArray(tags), Policies: policies}
 }
 
-type rollupTargetJSON struct {
-	Name     string          `json:"name"`
-	Tags     []string        `json:"tags"`
-	Policies []policy.Policy `json:"policies"`
+// RollupTargetView is a human friendly representation of a rollup rule target at a given point in time.
+type RollupTargetView struct {
+	Name     string
+	Tags     []string
+	Policies []policy.Policy
+}
+
+func newRollupTargetView(target RollupTarget) RollupTargetView {
+	return RollupTargetView{
+		Name:     string(target.Name),
+		Tags:     stringArrayFromBytesArray(target.Tags),
```

Comment:
nit: technically it's a slice not an array

Reply:
This naming convention was already being used.